pkg/validation/internal/bundle.go

```diff
@@ -133,23 +134,66 @@ func getOwnedCustomResourceDefintionKeys(csv *operatorsv1alpha1.ClusterServiceVe
 // - we could identify that the bundle size is close to the limit (bigger than 85%)
 func validateBundleSize(bundle *manifests.Bundle) []errors.Error {
 	warnPercent := 0.85
-	warnSize := int64(max_bundle_size * warnPercent)
+	warnSize := float64(max_bundle_size) * warnPercent
```

Comment:
It's a little hard to tell here, but I think `warnSize` will come out as a `float64`. Later we compare it to an `CompressedSize` an `int64`. Does this yield reliable results?

Reply:
We convert the compressedSize to float either see:`} else if float64(*bundle.CompressedSize) > warnSize {`

Also, we can check the tests so it shows fine: https://github.com/operator-framework/api/pull/229/files#diff-f4ded2e78dfeb583b32c94777fa213265c4748927288fb7c068d9e4b46e3fc5dR193-R197

Reply:
ah, sorry, I read into the deleted code to see the unconverted compare.

Reply:
That is fine. I. am closing this one as sorted out.